README.md

```diff
@@ -24,7 +24,7 @@ Example:
 ```bash
 python invoicex/main.py \
   --year-month 2022-04 \
-  --gh-user xmnlab \
+  --gh-user $USER \
```

Comment:
this is just an example, it is not a problem to have a fixed user here.
$USER is a system user, not a gh user, so it will be more confusing

Reply:
please check this comment